From: Hector Palacios <hector.palacios@digi.com>
Date: Tue, 1 Dec 2015 11:36:24 +0100
Subject: [PATCH] Use busfreq-imx6.h up to 3.15 kernel

For CC6 we're using Freescale 3.14.28 BSP. Even though we apply stability
patches that move the kernel version beyond 3.14.29 we still want to use
the GPU vivante external module build with the original BSP.

Signed-off-by: Hector Palacios <hector.palacios@digi.com>
---
 .../os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx6q14.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx6q14.c b/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx6q14.c
index 0eae8262c2ce..f166680affa2 100644
--- a/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx6q14.c
+++ b/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/platform/freescale/gc_hal_kernel_platform_imx6q14.c
@@ -74,7 +74,7 @@
 #include <linux/pm_runtime.h>
 #if LINUX_VERSION_CODE < KERNEL_VERSION(3, 10, 0)
 #include <mach/busfreq.h>
-#elif LINUX_VERSION_CODE < KERNEL_VERSION(3, 14, 29)
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(3, 15, 0)
 #include <linux/busfreq-imx6.h>
 #include <linux/reset.h>
 #else
